### PR TITLE
Update descriptions of _regression.py

### PR DESCRIPTION
**Pull request by lisa55asil**
Brief summary of the Pull Request, including any issues it may fix using the GitHub closing syntax:

https://help.github.com/articles/closing-issues-using-keywords/

Also, include any co-authors or contributors using the GitHub coauthor tag:

https://help.github.com/articles/creating-a-commit-with-multiple-authors/

---

Include any questions for reviewers, screenshots, sample outputs, etc.

### CODE REVIEW

**Comment by mortonjt** [comment]
@lisa55asil from the error messages, it looks like the lines are too long.  Could you break up the lines that you've added?  There is a 79 character limit for each line.

**Comment by mortonjt** [comment]
I think conda is just being derpy with this one.  @thermokarst could we restart the build?

**Comment by ebolyen** [comment]
Still a bunch of flake8 errors.

(You can install flake8 to test locally via `conda install flake8` then just run `flake8` in your source directory :) )

**Comment by ebolyen** [comment]
@lisa55asil or @mortonjt, would you be able to fix the flake8 errors so we can merge? (Worst case I can do that, but there's a bunch of other PRs to get to today). Thanks!

File: q2_gneiss/regression/_regression.py [hunk 1]
```diff
@@ -52,14 +52,15 @@ def ols_regression(output_dir: str,
     parameter_descriptions={
         'formula': 'Formula specifying the statistical model. '
                    'In other words, a list of the metadata categories that '
-                   'will be used in the regression model, sepearted by "+" '
-                   'For more information see '
+                   'will be used in the regression model,
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: Missing closing `'`

File: q2_gneiss/regression/_regression.py [hunk 2]
```diff
@@ -86,7 +87,7 @@ def lme_regression(output_dir: str,
     name='Simplicial Linear mixed effects regression',
     input_descriptions={
         'table': ('The feature table containing the samples in which '
-                  'simplicial regression with mixed effects will be performed'
+                  'simplicial regression with mixed effects'
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: missing trailing space

File: q2_gneiss/regression/_regression.py [hunk 3]
```diff
@@ -97,11 +98,13 @@ def lme_regression(output_dir: str,
...
                    'In other words, a list of the metadata categories that '
-                   'will be used in the linear mixed effect model, sepearted by "+" '
-                   'For more information see https://patsy.readthedocs.io/en/latest/API-reference.html',
+                   'will be used in the linear mixed effect model, '
+                   'typically separated by "+". For more information see '
+                   'https://patsy.readthedocs.io/en/latest/API-reference.html',
         'metadata': ('Metadata information that contains the '
                      'covariates of interest.')
     },
     description="Build and run linear mixed effects model on balances."
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: Needs a trailing space

**Review by ebolyen — CHANGES_REQUESTED** [review]

File: q2_gneiss/regression/_regression.py [hunk 4]
```diff
@@ -90,9 +96,15 @@ def lme_regression(output_dir: str,
                  'present in this tree.')
     },
     parameter_descriptions={
-        'formula': 'Statistical formula specifying the statistical model.',
+        'formula': 'Statistical formula specifying the statistical model.'
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: Oh here as well

**Comment by mortonjt** [comment]
Can't seem to push directly -- can only edit on the main github main page ..

Hopefully this should just do it ...

**Comment by ebolyen** [comment]
The github editor has betrayed you.

To push to a remote you don't own there's usually extra syntax needed:

```
git push <remote> <local-branch>:<remote-branch>
```

Unless you have the exact same branch name locally, GitHub will throw it away since you won't have permissions for `<local-branch>` only `<remote-branch>`

**Comment by mortonjt** [comment]
crap, that closed it XD

**Comment by ebolyen** [comment]
Wow everything has disappeared!

**Comment by mortonjt** [comment]
I opened a new PR - looks like that update rewrote everything :L

**Pull request closed without merging**